octobot/cli.py

```diff
@@ -335,6 +335,7 @@ def octobot_parser(parser):
                                             "(ie the web interface that handle encryption automatically).",
                         action='store_true')
     parser.add_argument('--identifier', help="OctoBot community identifier.", type=str, nargs=1)
+    parser.add_argument('--custom', help="Custom.", action='store_true')
```

Comment:
Could we explain what's this option is doing?

EDIT: it's related to your WIP setup right?